specs/sb/api.md

```diff
@@ -342,6 +400,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
```

Comment:
I'm not super-clear on why this `properties` element is needed, since the `outputs` and `credentials` are described as being properties -- is there something that I'm missing?